contracts/PartyBid.sol

```diff
@@ -412,26 +421,26 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
         _tokens = _value * TOKEN_SCALE;
     }
 
-    // ============ Internal: Bid ============
-
     /**
      * @notice The maximum bid that can be submitted
      * while leaving 5% fee for PartyDAO
      * @return _maxBid the maximum bid
      */
-    function _getMaximumBid() internal view returns (uint256 _maxBid) {
-        _maxBid = (totalContributedToParty * 100) / (100 + FEE_PERCENT);
+    function getMaximumBid() public view returns (uint256 _maxBid) {
+        _maxBid = (totalContributedToParty * 10000) / (10000 + ETH_FEE_BASIS_POINTS);
```

Comment:
_maxBid = (totalContributedToParty * (10000 - ETH_FEE_BASIS_POINTS)) / 10000;

Reply:
How to calculate _maxBid?

Reply:
maximum bid is the amount that can be bid and still pay the fee on the final bid to PartyDAO
let's say the final bid is 100 ETH and the fee is 5% of that, so 5 ETH. in this case the final amount spent is 105 ETH.
so, if the party has raised a total of 105 ETH, the maximum amount they can bid is 100 ETH
```
highest bid * 1.05 <= total amount contributed
```
so 
```
maximum bid = total amount contributed / 1.05
```

Reply:
OK, I understand it now.

Reply:
thank you for making this public!